src/modules/emote_menu/twitch/EmoteMenu.module.css

```diff
@@ -3,16 +3,18 @@
   position: relative !important;
 }
 
-.hideDefaultEmoteMenuButton {
-  button[data-a-target='emote-picker-button'] {
-    display: none !important;
-  }
-}
-
 .hideEmoteMenuButton {
   display: none;
 }
 
+.chatInputIcon {
+  display: inline-flex;
+  vertical-align: top;
```

Comment:
we should avoid changing container styles, especially because we're injecting blindly into a parent in the chain. if the button isn't rendering properly, we should change the button itself.